js/blocks/components/fields.js

```diff
@@ -95,27 +96,29 @@ const Fields = ( { fields, parentBlockProps, parentBlock, rowIndex } ) => {
 		 * If this is in a repeater row, the value is appropriate for that.
 		 *
 		 * @param {Object} props The properties of the Control function.
+		 * @param {Object} props.field The field.
+		 * @param {Object} props.parentBlockProps The props of the parent block.
+		 * @param {number} props.rowIndex The index of the repeater row (optional).
 		 */
-		const getValue = ( props ) => {
-			const { field, parentBlockProps, rowIndex } = props;
-			const attr = { ...parentBlockProps.attributes };
+		const getValue = ( {
+			field: ownField,
+			parentBlockProps: ownParentBlockProps,
+			rowIndex: ownRowIndex,
```

Comment:
These values, like `field` were declared in the parent scope. So to avoid naming collisions, this gives them local names like `ownField`.